backend/src/data/repository/user.ts

```diff
@@ -17,7 +17,8 @@ export class UserRepository {
   async createUser(newUser: DTO.IUser): Promise<DTO.IUserDoc> {
     const user = new User(newUser);
     const res = await user.save();
-    return res._id;
+    const friend = await UserRepository.addUserIdToFriends(res._id);
```

Comment:
why user is added to friends right after his creation? It should be done by a separate route, no?

Reply:
It just initialize of user, after creation we put record with userId and empty array of friends to collection friends. I'm not sure, that we should return user and create record in collection by using another service, it need for looking for userId in friends collection and put a new id of friend to array